app/models/request_issue.rb

```diff
@@ -15,6 +15,7 @@ class RequestIssue < ApplicationRecord
   validates :ineligible_reason, exclusion: { in: ["untimely"] }, if: proc { |reqi| reqi.untimely_exemption }
 
   enum ineligible_reason: {
+    duplicate_of_nonrating_issue_in_active_review: "duplicate_of_nonrating_issue_in_active_review",
     duplicate_of_issue_in_active_review: "duplicate_of_issue_in_active_review",
```

Comment:
might want to rename the tag here, and then add a task to maybe go migrate the values eventually.

```suggestion
    duplicate_of_rating_issue_in_active_review: "duplicate_of_issue_in_active_review",
```